src/foam/nanos/column/NestedPropertiesExpression.js

```diff
@@ -81,61 +81,118 @@ foam.CLASS({
         {
           name: 'i',
           class: 'Int'
+        },
+        {
+          name: 'expr',
+          javaType: 'foam.mlang.Expr'
         }
       ],
-      code: function (of, propName, i) {
-        if ( i === propName.length - 1 )
-          return of.getAxiomByName(propName[i]);
+      code: function (of, propName, i, expr) {
         var prop = of.getAxiomByName(propName[i]);
-        return foam.mlang.Expressions.create().DOT(prop, this.returnPropExpr(prop, this.returnDotExprForNestedProperty(prop.of, propName, ++i)));
+        if ( ! prop ) return null;
+
+        expr = this.buildPropertyExpr(prop, expr);
```

Comment:
Please create a new variable
```suggestion
        var propExpr = this.buildPropertyExpr(prop, expr);
```

Reply:
fixed. thanks